project-reports/2024/2024-annual-Hyperledger-Bevel.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: 2024 Annual Review Hyperledger Bevel
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
+
+[Hyperledger Bevel Operator Fabric LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel-operator-fabric)
+
+1. Getting code contribution on Bevel still remains a challenge, though the community usage and interest remains positive for both the projects as can be inferred from discussions and queries on discord channel and user participations on meetups.
+3. The projects have done well in terms of its release commitments.
+4. Bevel recently came out with its general availability release v1.0, with focus on seamless deployment. 
+5. Bevel-Operator-Fabric recently came out with its release v1.10.0, with a focus on stability of network deployment.
+6. Bevel has accepted a major code contribution. (helm charts for deploying Hyperledger Aries).
```

Comment:
Interesting stuff.  I looked a bit to see where this was and didn’t find, but I’m interested in this.  Where is the contribution coming from?  What does it cover?  Thanks!

Reply:
Hi Stephen, the contribution comes from user monar24 from superlogic team. It covers helm charts for deploying agents using ACP on Kubernetes. You can check the PR and follow the conversations around it [here](https://github.com/hyperledger/bevel/pull/2501)